src/components/20-molecules/dropdown/index.js

```diff
@@ -0,0 +1,224 @@
+import { LitElement, html, css, unsafeCSS, svg } from 'lit-element';
+import debounce from './utils/debounce';
+import styles from './index.scss';
+
+// TODO import icon from '@axa-ch/materials and extend them with classes'
+const ArrowIcon = svg`<svg class="a-icon__svg" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="32" height="32" viewBox="0 0 32 32">
+<path fill="currentColor" d="M16 28.067c-0.533 0-1-0.2-1.4-0.6l-14-14c-1.8-1.8 1-4.667 2.8-2.8l12.6 12.533 12.6-12.533c1.8-1.8 4.667 1 2.8 2.8l-14 14c-0.333 0.4-0.867 0.6-1.4 0.6z"></path>
+</svg>
+`;
+
+const DEFAULTS = {
+  isOpenClass: 'is-dropdown-open',
+  selectClass: 'js-dropdown__content',
+};
+
+class AXADropdown extends LitElement {
+  static tagName = 'axa-dropdown';
+  static styles = css`
+    ${unsafeCSS(styles)}
+  `;
+
+  static get properties() {
+    return {
+      items: { type: Array },
+      native: { type: Boolean },
+      size: { type: String },
+      title: { type: String },
+      value: { type: String },
+    };
+  }
+
+  firstUpdated() {
+    this.dropdown = this.shadowRoot.querySelector(`.${DEFAULTS.selectClass}`);
+    this.dropdownLinks = this.querySelectorAll('.js-dropdown__link');
+    window.addEventListener('resize', debounce(() => this.handleViewportCheck(this.dropdown), 250));
+  }
+
+  connectedCallback() {
+    super.connectedCallback();
+    this.isOpen = false;
+    this.selectedItem = this.items.filter(item => item.isSelected)[0] || null;
+    this.title = this.selectedItem.name;
+
+    window.openDropdownInstance = false;
```

Comment:
what if u have more then 1 dropdown on a page?

Reply:
plus we should use one window object that is genreic like: `__configs_plib`.

Reply:
example, every component does this in the constructor: `window._config_plig = window._config_plig || {};`

Reply:
once SSR is a topic, we also have to add something like: 

`const win = global || window`

Reply:
sure but add it when SRR is a topic. (over engineering)

Reply:
Yepp thats why i said: "once SSR is a topic"

Reply:
This 

> what if u have more then 1 dropdown on a page?

This exactly why we save the previous instance so we can close it on the next click.

Reply:
> example, every component does this in the constructor: `window._config_plig = window._config_plig || {};`

I agree, scope axa components within a sub object makes sense, something like `window.axaComponents` or window.axa.components` since components basically independent we should discuss the pros and cons.